<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="createListButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>400</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>to be deleted</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addItemButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Item</string>
    </property>
   </widget>
   <widget class="QPushButton" name="appendItemButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Append Item</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>20</y>
      <width>271</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="addressLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="findItemButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Item</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteItemButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Item</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteAllButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete All Items</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ageLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Age</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="findItemLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="editItemButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Item</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
